libsecp256k1.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "libsecp256k1",
+    "config-opts": [ "--disable-static" ],
+    "cleanup": [
+        "/lib/*.la",
+        "/lib/pkgconfig"
+    ],
+    "sources": [
+        {
+            "type": "git",
+            "url": "https://github.com/bitcoin-core/secp256k1",
+            "commit": "3a6fd7f636da5c6ee0702ab0ac40ff9f99753b7b"
```

Comment:
Is there anything special about this commit?

Reply:
No, it is simply fixed commit from May 20, 2020,
possibly more recent will be preferable, but this one I've tested.

Reply:
> but this one I've tested.

I've got file from Dash Electrum fork, and there is version from 2020-06-20.

Reply:
I'v can use 3f4a5a10e (current HEAD), and retest.

Reply:
Trying to build with 3f4a5a10e, will post info in about a hour.

Reply:
Please use the current HEAD. Thanks!

Reply:
added HEAD ref in 3bdca52

will build and test locally (test on new wallet creation lib is working), then post if ok